packages/salesforcedx-vscode-core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "salesforcedx-vscode-core",
-  "displayName": "Salesforce CLI Integration for Visual Studio Code",
+  "displayName": "CLI Integration",
```

Comment:
We should rename it Salesforce CLI Integrations since that's the official name for the cli.